src/tests/token-service/test-token-create-transaction.ts

```diff
@@ -1711,7 +1711,7 @@ describe("TokenCreateTransaction", function () {
     });
   });
 
-  describe("Expiration Time", () => {
+  describe.only("Expiration Time", () => {
```

Comment:
Intentional change?

Reply:
Sometimes I forget to remove those, which is like instructing the system to run only that specific test. I should probably implement Nathan's suggestion instead of Husky. This way, when I attempt to commit, it will prompt me to address these unnecessary items.